types/feathersjs__socketio/index.d.ts

```diff
@@ -1,12 +1,22 @@
 // Type definitions for @feathersjs/socketio 3.0
 // Project: https://feathersjs.com
 // Definitions by: Jan Lohage <https://github.com/j2L4e>
+//                 Desmond Koh <https://github.com/deskoh>
 // Definitions: https://github.com/feathersjs-ecosystem/feathers-typescript
 // TypeScript Version: 2.8
 
-/// <reference types="feathersjs__socket-commons"/>
-import SocketIO = require("socket.io");
+/// <reference types="node" />
 
-export default function feathersSocketIO(callback?: (io: SocketIO.Server) => void): () => void;
```

Comment:
Why did you remove these `export default` overloads in favor of an interface with an explicit `default` property? You should just be able to add `export const SOCKET_KEY: unique symbol;` and get the same effect.

Reply:
See discussion https://github.com/feathersjs-ecosystem/feathers-typescript/issues/29
and related PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34515.